src/main/resources/spark-swagger.conf

```diff
@@ -19,11 +19,11 @@ spark-swagger {
   # API related configs
 
   # Location where API docs will be published
-  host = "localhost:4567"
+  host = "w7tqwg7j5a.execute-api.us-east-1.amazonaws.com"
 
   # basepath for calling the API, if applicable, otherwise leave as "/"
   # as in https://example.api.com/<basepath>/user/all
-  basePath = "/"
+  basePath = "/dev1"
   docPath = "/doc"
```

Comment:
Does this make the `http://localhost:4567/doc.yaml` endpoint work? If so, can you add a comment about it?

Reply:
spark-swagger puts the auto-generated, non-edited yaml file at `http://localhost:4567/doc.yaml`, and that has nothing to do with the `host` or `basePath` attribute. (`host` or `basePath` are written into the document, so that readers can try out making API calls to that API Gateway similar to how they'd do it using tools such as Postman).